pay_android/pubspec.yaml

```diff
@@ -14,12 +14,12 @@
 
 name: pay_android
 description: A plug-in to add support for payments on the Android side of Flutter applications.
-version: 1.0.9
+version: 1.1.0
 homepage: https://github.com/google-pay/flutter-plugin
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Let's keep single quotes in these two lines.

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3